example.py

```diff
@@ -37,10 +37,16 @@ async def connect(args: argparse.Namespace, session: ClientSession) -> None:
     """Connect to the server."""
     websocket_url = args.url.replace("http", "ws") + "/api/websocket"
     async with HomeAssistantClient(websocket_url, args.token, session) as client:
-        await client.subscribe_events(log_events)
         # start listening will wait forever until the connection is closed/lost
-        await client.start_listening()
+        asyncio.create_task(log_exceptions(client.start_listening()))
```

Comment:
listener_task = asyncio.create_task(client.start_listening())
await client.subscribe_events(log_events)
await listener_task

Reply:
Yes, agree, that was what I meant in "we could also await for the created task also". :)